packages/react/src/components/AccountSettings/ConfigureMFA/MFAOption.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
This is the radio option for sms and totp, as shown inside in the radio button group here:
![image](https://user-images.githubusercontent.com/43682783/206542275-6dc3eaf0-3d40-4bc2-87fe-8b7ab90fbd19.png)